8 location codes

miklasr opened the pull request:
# Details

- Add Pages structure for documenting our standard
- Add initial content for the standard
    - Specifically, start documenting what particular codes mean

# Screenshots/GIF (if appropriate)

![image](https://user-images.githubusercontent.com/35376781/108710981-a39d8980-750c-11eb-8f20-638368892a78.png)

# Checklist

## Documentation
- [x] I have updated documentation appropriately (GitHub Pages)
- [x] I have tested the updated GitHub Pages locally to ensure the changes to documentation render correctly (see this [Guidance](https://ese-peasy.github.io/PosturePerfection/guidance.html) page)

Closes #8

Ashwin-MJ committed:
Run test suite using github actions

Ashwin-MJ committed:
Formatting fix

Ashwin-MJ committed:
GitHub Actions test

Ashwin-MJ committed:
Initial changes for UI

Ashwin-MJ committed:
Refactor into different components

Ashwin-MJ committed:
Added some placeholders for screen reading

Ashwin-MJ committed:
Acessibility fixes

Ashwin-MJ committed:
Add some tests

Ashwin-MJ committed:
Fix PR template

Ashwin-MJ committed:
Ensure compatability with smaller screen sizes

Commit:
Added content and initial pages for NMW standard

Commit:
Merge updates from main

Commit:
Add JQuery table for codes

Makes searching through codes easier

Commit:
Update definition of fields, extend location table

Ashwin-MJ reviewed (dismissed):
Nice! The suggested field widths make sense to me, and the table looks like a great starting point

Commit:
Update standard based on discussion

Ashwin-MJ reviewed (dismissed):
Looks like it hits everything we discussed earlier today, good stuff

Ashwin-MJ commented on docs/NMW-Standard/Location-Encoding.md:
```diff
@@ -0,0 +1,196 @@
+| Location attribute | Required? | Description |
+|--------------------|-----------|-------------|
+| NMW location code  | Yes       | The location code according to this standard |
+| Name               | Yes       | A name for the location. This could be, for example the name of a business |
```
Could we add one more required field `More Information` or something like that? This would be the more extensive audio snippet that is played to users if they tap the "more info" option. I guess this might be what you meant by the `Description` field, but in that case i suggest we make that field required. What do you think?

miklasr replied:
I've made the description field required so it needs to be present for a UUID mapping to be allowed

miklasr replied:
I think this conversation ca be resolved. Re-open if you think otherwise.

Commit:
Update standard based on discussion

- The location code has been shortened to max. 14 chars.
- §2.3 updated with character limits and changed required label

miklasr commented:
I have made minor updates based on our discussions:
- The UUID table now requires a description
- Fields now have character limits to ensure messages are short
    - Based on the focus group where succinctness was mentioned as important
- Shortened the NMW code so it now only occupies 14 characters

C-Begley commented on docs/NMW-Standard/Location-Encoding.md:
```diff
@@ -0,0 +1,198 @@
+
+## Outline
+
+Part of the NarrateMyWay project is concerned with the development of a language agnostic method for encoding locations. This also needs to be done in efficient way as some beacon technologies are limiting in the length of data that can be advertised to devices. Location encodings should also be useable in an off-line manner so travelling to a new location doesn't require using mobile data. Furth ...
```
"in an efficient way"

miklasr replied:
Thanks, fixed it

C-Begley commented on docs/index.md:
```diff
@@ -29,7 +29,12 @@ With a large array of navigation tools currently available, NarrateMyWay aims to
+## NMW Standard
+We have worked towards standardising the methods for interfacing with beacons used for audio navigation. This is intended to be compatible with the [Wayfindr](https://www.wayfindr.net/) open standard.
+
+Check out our documentation for the NMW expansion to the Wayfindr standard [here](/NMW-Standard).
```
Shouldn't these be NWM standard?

miklasr replied:
Would that be short for *Narrate Why May*?

C-Begley reviewed (dismissed):
Only two small points, but happy with it overall

Commit:
Grammar fix, spotted by @C-Begley
